python/baseline/utils.py

```diff
@@ -538,6 +551,12 @@ def write_json(content, filepath):
         json.dump(content, f, indent=True)
 
 
+@exporter
+def write_yaml(content, filepath):
```

Comment:
In `baseline.utils` there is now the `str_file` decorator that can handle opening and closing files for you. This lets your function transparently operate on either a file_path (str to a file) or a file like object while you as the programmer only have to write the code as if you are receiving an open file like object.

I think this makes code easier to write, easier to reuse, and more amenable to testing so I think we should start using it more.

This function can be rewritten to use this like so

```python
@str_file(file_path="w")
def write_yaml(content, file_path):
    yaml.dump(content, file_path)
```